bundles/CoreBundle/Resources/translations/de.json

```diff
@@ -647,6 +647,8 @@
     "resolver_strategy": "Resolver Strategy",
     "resolver_settings": "Resolver Settings",
     "csv_settings": "CSV Einstellungen",
+    "export_settings": "Export Einstellungen",
```

Comment:
Only `EN` translations are needed, the German translations are made here after this PR was merged: https://poeditor.com/join/project/VWmZyvFVMH please revert this changes :)